integrations/vite/index.test.ts

```diff
@@ -140,7 +125,10 @@ test(
 
     await process.onStdout((message) => message.includes('ready in'))
 
-    let css = await fetchCSS('/src/index.css', port)
+    let stylesheets = await fetchStylesheetsFromIndex(port)
+    expect(stylesheets).toHaveLength(1)
+    let [, css] = stylesheets[0]
```

Comment:
Not super ergonomic. I am wondering if i should just concatenate all css files into one string instead 🤔